src/test/datascience/notebook/helper.ts

```diff
@@ -597,18 +598,21 @@ async function getActiveInterpreterKernelConnection() {
         defaultNotebookTestTimeout,
         'Active Interpreter is undefined.2'
     );
+    const stopWatch = new StopWatch();
     return waitForCondition(
         () =>
             kernelFinder.kernels.find(
                 (item) =>
                     item.kind === 'startUsingPythonInterpreter' &&
                     areInterpreterPathsSame(item.interpreter.uri, interpreter.uri)
             ) as PythonKernelConnectionMetadata,
-        defaultNotebookTestTimeout,
+        defaultNotebookTestTimeout * 2,
```

Comment:
2 minutes is an enormous timeout, this will increase CI time when failures occur?